### PR TITLE
Update web_sql_injection_in_access_logs.yml

### PR DESCRIPTION
**Pull request by ThureinOo**
### Summary of the Pull Request
This PR updates `web_sql_injection_in_access_logs` with additional content

### Changelog
update: Detects sql injection exploitation attempts - Increase coverage

### Example Log Event
N/A

### Fixed Issues
N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/web/webserver_generic/web_sql_injection_in_access_logs.yml [hunk 1]
```diff
@@ -41,6 +41,13 @@ detection:
         - 'select%20*%20'
         - 'or 1=1#'
         - 'or%201=1#'
+        - 'concat_ws'
+        - 'table_names'
+        - 'column_names'
+        - 'schemata'
+        - 'database()'
+        - '</br>'
+        - '0x3a'
```
Comment (nasbench) [inline rules/web/webserver_generic/web_sql_injection_in_access_logs.yml:50]:
```suggestion
```

File: rules/web/webserver_generic/web_sql_injection_in_access_logs.yml [hunk 2]
```diff
@@ -41,6 +41,13 @@ detection:
         - 'select%20*%20'
         - 'or 1=1#'
         - 'or%201=1#'
+        - 'concat_ws'
```
Comment (nasbench) [inline rules/web/webserver_generic/web_sql_injection_in_access_logs.yml:44]:
```suggestion
        - 'concat_ws('
```

File: rules/web/webserver_generic/web_sql_injection_in_access_logs.yml [hunk 3]
```diff
@@ -41,6 +41,13 @@ detection:
...
+        - 'table_names'
+        - 'column_names'
+        - 'schemata'
```
Comment (nasbench) [inline rules/web/webserver_generic/web_sql_injection_in_access_logs.yml:47]:
```suggestion
```

File: rules/web/webserver_generic/web_sql_injection_in_access_logs.yml [hunk 4]
```diff
@@ -7,9 +7,9 @@ references:
     - https://www.acunetix.com/blog/articles/using-logs-to-investigate-a-web-application-attack/
     - https://brightsec.com/blog/sql-injection-payloads/
     - https://github.com/payloadbox/sql-injection-payload-list
-author: Saw Win Naung, Nasreddine Bencherchali (Nextron Systems)
+author: Saw Win Naung, Nasreddine Bencherchali (Nextron Systems), Thurein Oo (Yoma Bank)
```
Comment (nasbench) [inline rules/web/webserver_generic/web_sql_injection_in_access_logs.yml:10]:
```suggestion
author: Saw Win Naung, Nasreddine Bencherchali (Nextron Systems)
```

File: rules/web/webserver_generic/web_sql_injection_in_access_logs.yml [hunk 5]
```diff
@@ -41,6 +41,13 @@ detection:
         - 'select%20*%20'
         - 'or 1=1#'
         - 'or%201=1#'
+        - 'concat_ws'
+        - 'table_names'
+        - 'column_names'
+        - 'schemata'
+        - 'database()'
```
Comment (nasbench) [inline rules/web/webserver_generic/web_sql_injection_in_access_logs.yml:48]:
```suggestion
        - 'select database()'
        - 'select version()'
```

**Review by nasbench — APPROVED** [review]

File: rules/web/webserver_generic/web_sql_injection_in_access_logs.yml [hunk 6]
```diff
@@ -41,6 +41,9 @@ detection:
         - 'select%20*%20'
         - 'or 1=1#'
         - 'or%201=1#'
+        - 'concat_ws('
+        - 'select database()'
+        - 'select version()'
```
Comment (frack113) [inline rules/web/webserver_generic/web_sql_injection_in_access_logs.yml:46]:
```suggestion
        - 'select database()'
        - 'select version()'
        - 'select%20database()'
        - 'select%20version()'
```
to remain constant with the rest of the rule

Comment (nasbench) [reply rules/web/webserver_generic/web_sql_injection_in_access_logs.yml:46]: Thanks @frack113! Good additions

**Review by phantinuss — APPROVED** [review]

**Pull request merged**